content/en/blog/_posts/2023-08-22-sig-contribex-spotlight.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: blog
+title: "Spotlight on SIG ContribEx"
+date: 2023-08-22
+slug: sig-contribex-spotlight-2023
+author: Fyka Ansari
+---
+
+**Author**: Fyka Ansari
+
+Welcome to the world of Kubernetes and its vibrant contributor
+community! In this blog post, we'll be shining a spotlight on the
+[Special Interest Group for Contributor
+Experience](https://github.com/kubernetes/community/blob/master/sig-contributor-experience/README.md)
+(SIG ContribEx), an essential component of the Kubernetes project.
+
+SIG ContribEx in Kubernetes is responsible for developing and
+maintaining a healthy and productive community of contributors to the
+project. This involves identifying and addressing bottlenecks that may
+hinder the project's growth and feature velocity, such as pull request
+latency and the number of open pull requests and issues.
+
+SIG ContribEx works to improve the overall contributor experience by
+creating and maintaining guidelines, tools, and processes that
+facilitate collaboration and communication among contributors. They
+also focus on community building and support, including outreach
+programs and mentorship initiatives to onboard and retain new
+contributors.
+
+Ultimately, the role of SIG ContribEx is to foster a welcoming and
+inclusive environment that encourages contribution and supports the
+long-term sustainability of the Kubernetes project.
+
+In this blog post, [Fyka Ansari](https://twitter.com/1fyka) interviews
+[Kaslin Fields](https://twitter.com/kaslinfields), a DevRel Engineer
+at Google who serves as the Lead of SIG ContribEx and [Madhav
```

Comment:
```suggestion
at Google who serves as the Chair of SIG ContribEx and [Madhav
```
?

Reply:
I think that "... at Google, who is a Chair of SIG Contribex, and..." would work, no? I agree that "the Chair" should be avoided since it implies that there is a single one, when that's not true.

Reply:
I'm OK to merge this (and plan to); we can fix the specific wording with a couple of follow up PRs.

Reply:
```suggestion
at Google, who is a chair of SIG ContribEx, and [Madhav
```
adapted from https://github.com/kubernetes/website/pull/42522/files#r1292837546